GitUI/Script/ScriptOptionsParser.cs

```diff
@@ -349,7 +349,8 @@ private static string ParseScriptArguments(string argument, string option, IWin3
                         remote = selectedRemotes.Count == 1
                             ? selectedRemotes[0]
                             : AskToSpecify(selectedRemotes, revisionGrid);
-                        newString = module.GetSetting(string.Format(SettingKeyString.RemoteUrl, remote));
+                        url = module.GetSetting(string.Format(SettingKeyString.RemoteUrl, remote));
+                        newString = GetRemotePath(url);
```

Comment:
Can we add a test for this?

Reply:
I'll add some tests but not for every script option. (We should have asked for tests before the `ScriptOptionParser` was refactored. This is at least the second bug caused by this refactoring.)